packages/babel-helper-create-class-features-plugin/src/fields.ts

```diff
@@ -80,15 +84,18 @@ export function buildPrivateNamesNodes(
       init = t.callExpression(state.addHelper("classPrivateFieldLooseKey"), [
         t.stringLiteral(name),
       ]);
+      annotateAsPure(init);
+    } else if (privateFieldsAsSymbols) {
+      init = t.callExpression(t.identifier("Symbol"), [t.stringLiteral(name)]);
     } else if (!isStatic) {
       init = t.newExpression(
         t.identifier(!isMethod || isAccessor ? "WeakMap" : "WeakSet"),
         [],
       );
+      annotateAsPure(init);
```

Comment:
This is like before, but while at it, does `new WeakMap()` / `new WeakSet()` really need a PURE annotation?

Reply:
Oh actually, it looks like the `PURE` annotations is also needed for symbols:
![Terser playground with minifying `let x = /*#__PURE__*/ Symbol("x");let y = Symbol("y");`](https://user-images.githubusercontent.com/7000710/220083631-0b031cf1-f6b0-4de3-9439-240a41410c2d.png)

Reply:
Since this has the same effect as `new Symbol("y")`, I can see why.